explorer/explorermiddleware.go

```diff
@@ -35,6 +36,19 @@ func (exp *explorerUI) BlockHashPathOrIndexCtx(next http.Handler) http.Handler {
 				return
 			}
 		} else {
+                        //Handle blocks which are in future time
+                        maxHeight := exp.blockData.GetHeight()
+                        if height > int64(maxHeight) {
+                        maturityTime := int(float64(height - int64(maxHeight) ) * exp.ChainParams.TargetTimePerBlock.Minutes())
```

Comment:
This is doing: `int(float64(int64-int64) * time.Duration)`, where `int64` is the underlying type of `time.Duration`.  There is no need for the `float64` or `int` conversions.

Reply:
`maturityTime` is confusing.  Please call this `expectedTime`.